.github/workflows/reusable_ragger_tests_latest_speculos.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           - device: nanox
           - device: nanosp
           - device: stax
+          - device: flex
```

Comment:
Don't hardcode, use this: https://github.com/LedgerHQ/ledger-app-workflows/blob/master/.github/workflows/reusable_build.yml#L48
```
  call_get_app_metadata:
    # This job digests inputs and repository metadata provided by the `ledger_app.toml` manifest
    # file, in order to output relevant directories, compatible devices, and other variables needed
    # by following jobs.
    name: Retrieve application metadata
    uses: ./.github/workflows/_get_app_metadata.yml
    with:
      app_repository: ${{ inputs.app_repository }}
      app_branch_name: ${{ inputs.app_branch_name }}
      compatible_devices: ${{ inputs.run_for_devices }}
  ```